README.md

```diff
@@ -1,4 +1,7 @@
 # Nixtla &nbsp; [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Statistical%20Forecasting%20Algorithms%20by%20Nixtla%20&url=https://github.com/Nixtla/neuralforecast&via=nixtlainc&hashtags=StatisticalModels,TimeSeries,Forecasting) &nbsp;[![Slack](https://img.shields.io/badge/Slack-4A154B?&logo=slack&logoColor=white)](https://join.slack.com/t/nixtlacommunity/shared_invite/zt-1pmhan9j5-F54XR20edHk0UtYAPcW4KQ)
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-11-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
We should probably move this down with the other badges (line 17), because it currently looks like this:
![image](https://github.com/Nixtla/neuralforecast/assets/8473587/e7fdc8c0-533e-4d7b-b87a-cf8a6b2ff30d)

Reply:
tambien lo tenemos asi en stats, puede estar dificil porque lo pone automatico. lo pongo abajo entonces, arriba del rooster